pyvrp/cpp/search/bindings.cpp

```diff
@@ -207,8 +252,12 @@ PYBIND11_MODULE(_search, m)
                       &LocalSearch::neighbours,
                       &LocalSearch::setNeighbours,
                       py::return_value_policy::reference_internal)
-        .def_property_readonly("statistics",
-                               &LocalSearch::statistics,
+        .def_property_readonly("statistics", &LocalSearch::statistics)
```

Comment:
`py::return_value_policy::reference_internal` is gone here, is that intended?

Reply:
Ah yes, because `LS::statistics` creates a new object and does not need to take ownership.